src/lv_core/lv_obj.h

```diff
@@ -446,6 +454,13 @@ void lv_obj_set_top(lv_obj_t * obj, bool en);
  */
 void lv_obj_set_drag(lv_obj_t * obj, bool en);
 
+/**
+ * Set the directions an object can be dragged in
```

Comment:
It should state that "OR"ed values of `lv_drag_dir_t` can be used as `drag_dir`.